python/lsst/jointcal/jointcal.py

```diff
@@ -296,9 +344,15 @@ def __init__(self, *, config=None):
         if not config.doAstrometry:
             self.prerequisiteInputs.remove("astrometryRefCat")
             self.outputs.remove("outputWcs")
+            for key in list(self.outputs.keys()):
+                if "astrometry" in key:
```

Comment:
Minor thing, but both here and below, I can imagine someone putting in an output that has the word "astrometry" or "photometry" in it and being surprised when it's stripped.  Is it worth being even more explicit with "metricvalue_jointcal_astrometry" and "metricvalue_jointcal_photometry"?